src/refresh/vkpt/main.c

```diff
@@ -537,24 +545,30 @@ static bool pick_surface_format_hdr(VkSurfaceFormatKHR* format, const VkSurfaceF
 		for(int j = 0; j < num_avail_surface_formats; j++) {
 			if((acceptable_formats[i].format == avail_surface_formats[j].format)
 				&& (acceptable_formats[i].colorSpace == avail_surface_formats[j].colorSpace)){
-				*format = avail_surface_formats[j];
+				picked_fmt->surface_fmt = avail_surface_formats[j];
+				picked_fmt->swapchain_view_fmt = avail_surface_formats[j].format;
 				return true;
 			}
 		}
 	}
 	return false;
 }
 
-static bool pick_surface_format_sdr(VkSurfaceFormatKHR* format, const VkSurfaceFormatKHR avail_surface_formats[], size_t num_avail_surface_formats)
+static bool pick_surface_format_sdr(picked_surface_format_t* picked_fmt, const VkSurfaceFormatKHR avail_surface_formats[], size_t num_avail_surface_formats)
 {
-	VkFormat acceptable_formats[] = {
-		VK_FORMAT_R8G8B8A8_SRGB, VK_FORMAT_B8G8R8A8_SRGB,
+	struct {
+		VkFormat format;
+		VkFormat swapchain_view_fmt;
+	} acceptable_formats[] = {
+		{VK_FORMAT_R8G8B8A8_SRGB, VK_FORMAT_R8G8B8A8_SRGB}, {VK_FORMAT_B8G8R8A8_SRGB, VK_FORMAT_B8G8R8A8_SRGB},
+		{VK_FORMAT_R8G8B8A8_UNORM, VK_FORMAT_R8G8B8A8_SRGB}, {VK_FORMAT_B8G8R8A8_UNORM, VK_FORMAT_B8G8R8A8_SRGB},
```

Comment:
Suggest splitting these lines so that there is one entry per line.

Reply:
Sure